napari_allencell_annotator/util/file_utils.py

```diff
@@ -51,10 +66,18 @@ def get_sorted_files_in_dir(dir_path: Path) -> list[Path]:
         dir_path: list[Path]
             The path to a directory
         """
-        return sorted(list(dir_path.glob("*.*")))
+
+        return sorted(list(dir_path.glob("*")))
 
     @staticmethod
     def shuffle_file_list(files: list[Path]) -> list[Path]:
         shuffled_list = files.copy()
         random.shuffle(shuffled_list)
         return shuffled_list
+
+    @staticmethod
+    def get_file_name(path: Path):
+        if path.suffix == ".zarr":
+            return path.parent.stem
+        else:
+            return path.name
```

Comment:
test for this method

Reply:
and comment

Reply:
tested in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/4d85d51f6e27a33ddebe670b1a8fd12e2b38bb3c